### DISCUSSION TITLE
/bin/sh: lsb_release: command not found

### DISCUSSION THREAD

**Discussion by toddgeist** [vercel/community#4791] [Help]
### Summary

Vercel logs are showing this error in production.

`/bin/sh: lsb_release: command not found`

Why?  What is this?

### Steps to Reproduce

This is in a production app.

**Comment by amyegan** [comment]
Is that error showing in the build log, or somewhere else?

**Comment by toddgeist** [comment]
it  is showing in vercel Logs. 
![TG2023-11-09 at 13 45 45](https://github.com/vercel/community/assets/316792/24dddc2a-d492-4cac-94b6-77ea46b7104c)
